Ogury/OguryAdapterConfiguration.m

```diff
@@ -0,0 +1,80 @@
+//
+//  Copyright © 2019 Ogury Ltd. All rights reserved.
+//
+
+#import "OguryAdapterConfiguration.h"
+#import <OguryAds/OguryAds.h>
+
+#if __has_include("MoPub.h")
+    #import "MPLogging.h"
+#endif
+
+#pragma mark - Constants
+
+NSString * const kOguryConfigurationAdUnitId = @"ad_unit_id";
+NSString * const kOguryErrorDomain = @"com.mopub.mopub-ios-sdk.mopub-ogury-adapters";
+static int const OguryMissingAssetKeyErrorCode = 2006;
+
+static NSString * const OguryConfigurationMediationName = @"MoPub";
+static NSString * const OguryConfigurationAdapterVersion = @"2.2.4.0";
+static NSString * const OguryConfigurationNetworkName = @"ogury";
+
+static NSString * const OguryConfigurationKeyAssetKey = @"asset-key";
+
+@implementation OguryAdapterConfiguration
+
+#pragma mark - Properties
+
+- (NSString *)adapterVersion {
+    return OguryConfigurationAdapterVersion;
+}
+
+- (NSString *)biddingToken {
+    return nil;
+}
+
+- (NSString *)moPubNetworkName {
+    return OguryConfigurationNetworkName;
+}
+
+- (NSString *)networkSdkVersion {
+    return [[OguryAds shared] sdkVersion];
+}
+
+#pragma mark - Methods
+
++ (void)updateInitializationParameters:(NSDictionary *)parameters {
```

Comment:
We cannot update the asset key in the SDK so I think we can keep this method empty.

Reply:
Just checked the adaptor creation guidelines, I'll empty it.